src/client/components/app/Transactions/TestTx.tsx

```diff
@@ -35,12 +31,14 @@ export const TestTx: FC<TestTxProps> = ({ onClose, children, ...props }) => {
 
         <TxActionButton onClick={() => console.log('deposit')}>Deposit</TxActionButton>
 
-        <TxActionButton onClick={() => console.log('withdraw')} contrast></TxActionButton>
+        <TxActionButton onClick={() => console.log('withdraw')} contrast>
+          Loading
```

Comment:
should we have this localized?

Reply:
This is for dev testing only. No for users